src/main/java/bio/terra/service/snapshotbuilder/utils/CriteriaQueryBuilder.java

```diff
@@ -43,7 +44,12 @@ private TablePointer getRootTablePointer() {
   private FieldVariable getFieldVariableForRootTable(String columnName) {
     return new FieldVariable(new FieldPointer(getRootTablePointer(), columnName), rootTable);
   }
-
+  /**
+   * Generate a filter for a SnapshotBuilderProgramDataRangeCriteria
+   * <pre>{@code
+   * p.year_of_birth >= low_value AND p.year_of_birth <= high_value
+   * </pre>
+   */
```

Comment:
I know I previously advocated for SQL comments, but per our team conversation yesterday, I think we might actually want to remove these and instead make a test that shows the generated SQL.

Reply:
They're removed!

Reply:
Okay, great and there tests that still represent the same info already?

Reply:
Okay great and there are already tests that represent that same info, right?

Reply:
Ya, I honestly made the comments from the tests!